package.json

```diff
@@ -31,10 +31,10 @@
     "start": "vite",
     "verify-backend": "./scripts/verify-backend.sh",
     "storybook": "storybook dev -p 6006",
-    "storybook:build": " . ./scripts/app-version.sh && env-cmd -e local -r .env-cmdrc.local.json storybook build",
+    "storybook:build": "./scripts/deploy/app-version.sh && env-cmd -e local -r .env-cmdrc.local.json storybook build",
     "test": "jest --watchAll=false",
     "test:watch": "jest --watch --verbose",
-    "upload-source-maps": "node ./scripts/upload-source-maps.js",
+    "upload-source-maps": "node ./scripts/deploy/upload-bugsnag-sourcemaps.js",
```

Comment:
im not sure if it matters but in Spruce it says:
>`"upload-source-maps": "env-cmd -e production node ./scripts/deploy/upload-bugsnag-sourcemaps",`

wanted to point it out just in case